coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/person/ProposalBudgetProjectPersonnelController.java

```diff
@@ -175,6 +175,7 @@ private void setBudgetPeriodStartDateAndEndDateOnLineItems(ProposalBudgetForm fo
 		form.getAddProjectPersonnelHelper().getBudgetLineItem().setEndDate(budgetPeriod.getEndDate());
 		form.getAddProjectPersonnelHelper().getBudgetPersonnelDetail().setStartDate(budgetPeriod.getStartDate());
 		form.getAddProjectPersonnelHelper().getBudgetPersonnelDetail().setEndDate(budgetPeriod.getEndDate());
+		form.getAddProjectPersonnelHelper().getBudgetPersonnelDetail().setBudgetId(form.getBudget().getBudgetId());
```

Comment:
Ok. Will update to set budget while invoking add personnel and budget id will be set in setBudget